playbook/lib/playbook/bottom.rb

```diff
@@ -24,9 +24,12 @@ def bottom_props
 
       selected_props.map do |k|
         value = send(k)
-        return nil unless bottom_values.include? value
-
-        "bottom_#{value}"
+        css = if value.is_a?(Hash) && value[:inset] && bottom_values.include?(value[:value])
+                "bottom_#{value[:value]}_inset"
+              elsif bottom_values.include?(value)
+                "bottom_#{value}"
+              end
+        css
```

Comment:
Akin to what @jasperfurniss mentioned above, I think this can be abstracted away to `playbook/position.rb` as a single method for top, left, bottom, and right.

Reply:
Done! Thank you for the feedback, definitely a lot better than the repetition that was there before @thestephenmarshall